app/Console/Commands/BackupDatabase.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class BackupDatabase extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:backup-database';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command description';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+{
+    $databaseName = env('DB_DATABASE');
```

Comment:
i have to tell you a little note here so when you are working in project you have to consider this as

> [!WARNING]
> if you execute the config:cache command during your deployment process, you should be sure that you are only calling the env function from within your configuration files. Once the configuration has been cached, the .env file will not be loaded; therefore, the env function will only return external, system level environment variables.

this is what is common done in production to optimize the application they run commands to cache values, views or configuration so as to not see uncommon behavior in your application here is some recommendations for you

- use `env()` function only in configuration files, these are the files found in `config/`
- in any place other than your config files use `config()` function

I know the website tell you to use `env()` function but this is common in websites that gives you a solution, sometimes it doesn't tell you the whole story